facebook/__init__.py

```diff
@@ -249,6 +249,7 @@ def request(
             raise GraphAPIError(response)
 
         headers = response.headers
+        result = {}
```

Comment:
Is this necessary? If we get down to `if result and isinstance(result, dict) and result.get("error"):`, shouldn't `result` always be defined?